webview/src/ConfigEditor/fields.tsx

```diff
@@ -109,8 +109,21 @@ export function passphrase(config: FileSystemConfig, onChange: FSCChanged<'passp
   return <FieldDropdownWithInput key="passphrase" label="Passphrase" {...{ value, values, description }} onChange={callback} optional={true} />
 }
 
+export function debugPort(config: FileSystemConfig, onChange: FSCChanged<'debugPort'>): React.ReactElement {
+  const callback = (value?: number) => onChange('debugPort', value);
+  const description = 'Debug port to attach';
+  return <FieldNumber key="debugPort" label="Remote Debugging Port" value={config.debugPort} onChange={callback} optional={true} description={description} />  
+}
+
+export function debugPreLaunch(config: FileSystemConfig, onChange: FSCChanged<'debugPreLaunch'>): React.ReactElement {
+  const callback = (value?: string) => onChange('debugPreLaunch', value);
+  const description = 'Task to run before debug session starts. SSH command: e.g. `/opt/vistec/python/bin/python -m ptvsd --host ${config.host} --port ${config.debugPort} --wait ${file}`';
```

Comment:
A bit of a long description which doesn't specify that the command will be run _locally_